installer/tue-install-impl.bash

```diff
@@ -839,6 +886,8 @@ do
     fi
 done
 
+# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
```

Comment:
Why these here? The script was pretty clear without these separators. This is just making it ugly.

Reply:
My opinion was that the main loop needed a better structure. I converted the separation lines to double empty lines.